src/lib/holocene/combobox/combobox.svelte

```diff
@@ -364,7 +364,6 @@
         closeList();
         break;
       case 'ArrowDown':
-        openList();
```

Comment:
With the addition of [this line of code](https://github.com/temporalio/ui/pull/2431/files#diff-a3db663fd01dcc941f639c835078a17e5ca9d85a8c6be95c615f59355555d827R171) we’re clearing the `filterValue` on `openList()`. 

I think it's safe to remove `openList()` from `ArrowDown` here. All it will change is that a user can't hit `ESC` and then `ArrowDown` to open the menu again.